src/automation/auto_inventory.py

```diff
@@ -0,0 +1,108 @@
+'''
+Script for automated running of inventory in the AWS S3 bucket
+Allows for configuration of categories of data to run (currently referencing atm_dicts), an end date, and number of days ago
+If nothing is specified in the configuration, it runs the full period of time for all atmosphere variables. 
+Runs in parallel on given number of CPUs. 
+'''
+import automation_utils as au
+import atm_dicts
+import yaml_generation as yg
+
+import argparse
+from joblib import Parallel, delayed
+import os
+
+from datetime import datetime as dt
+from datetime import timedelta
+from pandas import Timestamp
+
+#argparse section
+parser = argparse.ArgumentParser()
+parser.add_argument("-cat", dest="category", help="Category of variables to inventory. Valid options: atmosphere", choices=['atmosphere'], default="atmosphere", type=str)
+parser.add_argument("-end", dest="end_date", help=f"End date to use for run. Format expected {au.DATESTR_FORMAT}. If not provided, uses the current time.", type=str)
+parser.add_argument("-ago", dest="days_ago", help="Number of days ago to run the inventory for. If provided, must be positive integer. If not provided, it will run the full extent of the inventory.", default=0, type=int)
```

Comment:
```suggestion
parser.add_argument("-ago", dest="days_ago", help="Number of days before today or a given "end_date" (defined by the -end argument) over which to run the inventory. If provided, must be positive integer. If not provided, it will run the full extent of the inventory.", default=0, type=int)
```

Reply:
my suggestion above is based on my understanding - please adjust accordingly if I am mistaken